client/src/contexts/Contexts.tsx

```diff
@@ -21,7 +21,7 @@ ProductStyleContext.displayName = 'ProductStyleContext';
 const RelatedProducts = createContext<number[]>(undefined);
 RelatedProducts.displayName = 'RelatedProducts';
 
-const ReviewsContext = createContext<reviewsProps>(undefined);
+const ReviewsContext = createContext<reviews>(undefined);
```

Comment:
I agree that a better name could be used here, however "reviews" is already a different interface being exported from the file reviews.interface.ts. We would either need to make a sperate name , maybe "reviewsProvider", or move this interface to a separate file.